Tests/FoundationInternationalizationTests/Formatting/Number/BinaryInteger+FormatStyleTests.swift

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2023 Wade Tregaskis
```

Comment:
The copyright should be assigned to the Swift project. While we're at it, updating the copyright years of the files being modified would be welcome.

Reply:
Done.